src/Components/Web.JS/src/InputFile.ts

```diff
@@ -86,6 +89,7 @@ async function toImageFile(elem: InputElement, fileId: number, format: string, m
     contentType: format,
     readPromise: undefined,
     arrayBuffer: undefined,
+    blob: undefined,
```

Comment:
I suspect that, in declaring this, you might be breaking the intent of the code below (starting at line 97):

```js
// Attach the blob data itself as a non-enumerable property so it doesn't appear in the JSON.
Object.defineProperty(result, 'blob', { value: resizedImageBlob });
```

Since you're making it an explicit property here, it will start appearing on the JSON.

That might no longer matter if we're no longer JSON serializing it anywhere. If that is the case, could we eliminate the whole `defineProperty` thing? But if we are still JSON serializing it, then I guess it needs not to be declared explicitly like this.

Reply:
Thanks for pointing this out, that has now been removed with https://github.com/dotnet/aspnetcore/pull/33900/commits/7d4f8f8ccacaf243be493281e6cee566cae0deef. We do now serialize it for the `ToImageFile` endpoint:

https://github.com/dotnet/aspnetcore/blob/44253f8f56cd6caf44328a8ab3b385664d819a93/src/Components/Web.JS/src/InputFile.ts#L59

https://github.com/dotnet/aspnetcore/blob/44253f8f56cd6caf44328a8ab3b385664d819a93/src/Components/Web/src/Forms/InputFile.cs#L91

But it's not really a concern as the `Blob` serializes to `{}` by default (which is dropped by C#), so we aren't adding any significant overhead to the interop.